org.eclipse.xtext.p2repository/category.xml

```diff
@@ -1,336 +1,5 @@
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.codemining" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.codemining.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.codetemplates" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.codetemplates.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.codetemplates.ide" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.codetemplates.ide.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.codetemplates.ui" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.codetemplates.ui.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.ecore" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.ecore.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.shared" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.ui.shared.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xbase.junit" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xbase.junit.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xbase.ui.testing" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xbase.ui.testing.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xbase.testlanguages.ui" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xbase.testlanguages.ui.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xbase.ui" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xbase.ui.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xtext.ui" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xtext.ui.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xtext.ui.examples" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xtext.ui.examples.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xtext.ui.graph" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-	<bundle id="org.eclipse.xtext.xtext.ui.graph.source" version="0.0.0">
-		<category name="Xtext" />
-	</bundle>
-
    <feature id="org.eclipse.xtext.sdk" version="0.0.0">
```

Comment:
is the 0.0.0 intentional. before we use x.x.xqualifer here too

Reply:
Yes, it is intentional: specifying the version for our bundles/features doesn't make much sense and would force us to keep them consistent.